views/js/review/runner.js

```diff
@@ -0,0 +1,221 @@
+    'use strict';
+
+    /**
+     * Builds a test runner to preview test
+     * @param {jQuery|HTMLElement|String} container - The container in which renders the component
+     * @param {Object}   config - The testRunner options
+     * @param {Object[]} [config.providers] - A collection of providers to load.
+     * @param {Boolean} [config.replace] - When the component is appended to its container, clears the place before
+     * @param {Number|String} [config.width] - The width in pixels, or 'auto' to use the container's width
+     * @param {Number|String} [config.height] - The height in pixels, or 'auto' to use the container's height
+     * @param {Boolean} [config.options.fullPage] - Force the previewer to occupy the full window.
+     * @param {Booleanh} [config.options.readOnly] - Do not allow to modify the previewed item.
+     * @param {Function} [template] - An optional template for the component
+     * @returns {previewer}
+     */
+    return function reviewFactory(container, config = {}, template = null) {
+        const defaultPlugins = [{
+            module: 'taoReview/review/plugins/navigation/next-prev-review/next-prev-review',
+            bundle: 'taoReview/loader/qtiReview.min',
+            category: 'navigation'
+        }];
+
+        const testRunnerConfig = {
+            testDefinition: 'test-container',
+            serviceCallId: 'previewer',
+            proxyProvider: 'qtiItemReviewProxy',
+            bootstrap : {
+                serviceController : 'Review',
+                serviceExtension : 'taoReview'
+            },
+            // loadFromBundle: 'true',
+            providers: {
+                runner: {
+                    id: 'qtiItemReview',
+                    module: 'taoReview/review/provider/item/item',
+                    bundle: 'taoReview/loader/qtiReview.min',
+                    category: 'runner'
+                },
+                proxy: {
+                    id: 'qtiItemReviewProxy',
+                    module: 'taoReview/review/proxy/item',
+                    bundle: 'taoReview/loader/qtiReview.min',
+                    category: 'proxy'
+                },
+                communicator: {
+                    id: 'request',
+                    module: 'core/communicator/request',
+                    bundle: 'loader/vendor.min',
+                    category: 'communicator'
+                },
+                plugins: defaultPlugins
+            },
+            options: {
+                readOnly: config.readOnly,
+                fullPage: config.fullPage,
+            }
+        };
+
+        const testData = {
+            config: {
+                allowShortcuts: true,
+                shortcuts: {
+                    'next-prev-review': {
+                        next: 'arrowright',
+                        prev: 'arrowleft'
+                    }
+                },
+            }
+        };
```

Comment:
I understand this is temporary, but wouldn't it be better to have such mock data at least in a separate file?
And then it would have been better to set the data from the controller, and/or from the proxy.